docs/feature-flagging/experiment-assignment.md

```diff
@@ -23,3 +23,51 @@ Click "Create Experiment" on the experiment allocation to create an experiment.
 Once assignments have begun, they will need to be logged to your data warehouse to be analyzed by Eppo. All SDKs take in a callback function that is called with the assignment parameters (`subject_id`, `experiment_id`, `timestamp`, etc.) passed in when assignments are made. That function is completely managed by you - the only requirement is that the assignment data eventually makes it to your warehouse. 
 
 Congratulations on setting up your first Eppo randomized experiment! Refer to our [analysis setup guide](/experiment-quickstart) to connect Eppo to your data warehouse and easily analyze the results.
+
+## Running no/low code experiments in Eppo
+
+Often times there is a need for teams to be able to release changes quickly without a code release, or there are frequently tested parts of a site or app where teams would like to execute changes in a self-service manner. Running experiments without having to release additional code is easy to do in Eppo by using a JSON flag type to parameterize parts of the site or app where frequent changes are made and creating additional experiment allocations on the same flag.
+
+### Creating multiple experiments on the same flag
+
+For this example let’s use the following feature flag in Eppo. Right now there are 2 allocations, one is a feature gate that prioritizes sending any internal users to the treatment variation, and the other is an experiment allocation that evaluates whether the other users meet the audience criteria, and if they do split them evenly between the control and treatment variations on 50% of the total traffic.
+
+![Initial Allocation set up with one feature gate and one experiment](/img/feature-flagging/experiment-assignment/initial-experiment-allocation.png)
+
+Before creating another experiment allocation on this flag, create a new variation by editing your feature flag and clicking ‘Add Variation’.
+
+![Adding a variation to a feature flag](/img/feature-flagging/experiment-assignment/add-variation.png)
+
+Once the new variation is added, go back to the feature flag overview page and create a new experiment allocation by clicking ‘Add Allocation’ and selecting ‘Experiment'.
+
+After adding an additional Experiment Allocation, the allocation shows up on your Feature Flag overview page.
+
+![Allocation setup with additional experiment](/img/feature-flagging/experiment-assignment/additional-experiment-allocation.png)
+
+Right now this allocation logic splits the remaining users 50/50 into the second and third variations if those users do not qualify for the first 2 allocations. If one of the other experiments has been complete, the allocation can be archived by selecting the ‘bin’ icon on the right hand side of the allocation. Additionally, by clicking the gray side bar on the right hand side of the allocation, allocations can be dragged and dropped to reprioritize the order of logic executed.
```

Comment:
I don't think these 3 sentences belong together in the same paragraph. The first one is good and in flow. Archiving I would put at the very end. The note about reordering would be more helpful if it gave an example of _why_ reordering might be useful, and also some warning on the downstream impact (a running experiment might get affected/invalidated if an allocation is dropped above it).

Reply:
Updated and added warning